PaymentServiceSample/src/main/java/com/aevi/sdk/pos/flow/paymentservicesample/service/PaymentService.java

```diff
@@ -86,6 +92,20 @@ protected void subscribeToFlowServiceEvents(BaseStageModel model, Class<? extend
         });
     }
 
+    private void handleCancelOrResume(BaseStageModel model, Class<? extends Activity> activityToRestart) {
+        // For illustration, cancel immediately for TRANSACTION_PROCESSING but resume for others
+        if (model instanceof TransactionProcessingModel) {
```

Comment:
Should we provide a nice easy way to get whether or not it was a cancel or resume?

Reply:
Hmm...what do you mean? `cancelOrResume` is a new event, `resume` remains as before

Reply:
updated

Reply:
Lovely